docs/Code Exposure.md

```diff
@@ -0,0 +1,261 @@
+[AWS RE:INFORCE](https://reinforce.awsevents.com/faq/) \
+[Self-Service Security Assessment](https://aws.amazon.com/blogs/publicsector/assess-your-security-posture-identify-remediate-security-gaps-ransomware/)
+>>>
+
+- `Which roles are able to make changes to services within your account?`
+- `Which users have those roles assigned to them? Is least privilege being followed, or do super admin users exist?`
+- `Has a security assessment been performed against your environment? Do you have a known baseline to detect "new" or "suspicious" things?`
+
+### Communication Technology
+- `What technology is used within the team/company to communicate issues? Is there anything automated?`
+>>>
+Telephone \
+E-mail \
+SMS \
+AWS SES \
+AWS SNS \
+Slack \
+Chime \
+Teams \
+Other? 
+>>>
+
+## Detection
+
+### Logging and S3 Bucket Checks
+- Ensure CloudTrail is enabled in all regions: `./prowler -c check21`
+- Ensure CloudTrail log file validation is enabled: `./prowler -c check22`
+- Ensure S3 bucket access logging is enabled on the CloudTrail S3 bucket: `./prowler -c check26`
+- Ensure there are no S3 buckets open to Everyone or Any AWS user: `./prowler -c extra73`
+- Identify the resources in your organization and accounts; such as Amazon S3 buckets or IAM roles; that are shared with an external entity: `./prowler -c extra769`
+- Find resources exposed to the internet: `./prowler -g group17`
+
+## Escalation Procedures
+- `Who is monitoring the logs/alerts, receiving them and acting upon each?`
+- `Who gets notified when an alert is discovered?`
+- `When do public relations and legal get involved in the process?`
+- `When would you reach out to AWS Support for help?`
+
+## Analysis
+It is highly recommended to export logs to a security incident event management (SIEM) solution (such as Splunk, ELK stack, etc.) to aid in viewing and analyzing a variety of logs for a more complete attack timeline analysis.
+
+### CloudTrail: Public S3 Bucket
+By default, CloudTrail logs API calls that were made in the last 90 days, but not log requests made to objects. You can see bucket-level events on the CloudTrail console. However, you can't view data events (Amazon S3 object-level calls) there—you must parse or query CloudTrail logs for them. 
+
+1. Navigate to your [CloudTrail Dashboard](https://console.aws.amazon.com/cloudtrail)
+1. In the left-hand margin select `Event History`
+1. In the drop-down change from `Read-Only` to `Event Name`
+1. Review CloudTrail logs for the eventnames `GetPublicAccessBlock` and `DeletePublicAccessBlock`
+
+### CloudTrail: Public S3 Object
+You can also get CloudTrail logs for object-level Amazon S3 actions. To do this, enable data events for your S3 bucket or all buckets in your account. When an object-level action occurs in your account, CloudTrail evaluates your trail settings. If the event matches the object that you specified in a trail, the event is logged. 
```

Comment:
Recommend adding the process to enable events OR linking "enable data events" to the process in our documentation so it's clear how to do so

Reply:
Added recommended link.